css/style.css

```diff
@@ -904,6 +2209,61 @@
   .arrow {
     padding-left: 2px;
   }
+
+  .wrap-information {
+    display: flex;
+    justify-content: flex-end;
+    flex-direction: column;
+    background-position-y: top;
+  }
+
+  .info__title {
+    display: none;
+  }
+
+  .red-first, 
+  .details {
+    max-width: 100%;
+    float: none;
+  }
+
+  .icons {
+    display: none;
+  }
+
+  .info-order {
+    display: none;
+  }
+
+  .title {
+    white-space: nowrap;
+  }
+
+  .rating-container {
+    display: none;
+  }
+
+  .description__rating {
+    display: none;
+  }
+
+  .images {
+    position: relative;
+  }
+
+  .wrap__img, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)